src/quo/core.cljs

```diff
@@ -232,6 +233,7 @@
 ;;;; Dropdowns
 (def dropdown quo.components.dropdowns.dropdown.view/view)
 (def network-dropdown quo.components.dropdowns.network-dropdown.view/view)
+(def dropdown-input quo.components.dropdowns.dropdown-input.view/view)
```

Comment:
Can we keep this alpha sorted, makes this file easier to keep organised/ check what's there 👍